modules/mobile/spec/request/v1/immunizations_request_spec.rb

```diff
@@ -0,0 +1,538 @@
+# frozen_string_literal: true
```

Comment:
This is mostly just a copy of the v0 version of the tests. Because v0 is being deprecated, those tests shouldn't be changing in the future, so I didn't want to create any shared tests.